README.md

```diff
@@ -1,5 +1,61 @@
-# curriculum-cr-training-contact-form
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This repo host an initial code for the tasks performed during the Code Reviewers training.
+# Portfolio
 
-Application is based on this [Figma design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471).
+This project is part of the Microverse curriculum for software development and it's meant to be a template for our own portfolio later on.
+Here you will find some projects that I've been doing in the past as well as a contact form, an about section to get to know me a bit better and a list of my skills. Make sure to check it out!
+
+## Built With
+
+- HTML, CSS, JavaScript
+- Linters used: ESLint, Stylelint and webhint.
+- No frameworks, just plain JavaScript.
+
+## Getting Started
+
+**To get started with this project, all you have to do is cloning it!**
+
+If you want to see this repo in your local machine, you have two option:
+1. You can clone it to your computer and run it with live server or a localhost.
+2. You can check at this link: [Portfolio](https://thisisjustafakelink.com)
+
+### Prerequisites
+
+You don't need any prerequisites to use this project, just enjoy!
+
+### Install
+
+No installation is needed, just clone the project (or open it in the browser using the link below) and open it in your favorite text editor or browser.
+
+### Usage
+
+To use this project, you can just navigate to the file and open the index.html file with your favorite browser, either chrome or firefox.
+
+### Run tests
+
+No tests were written for this project, but you can make your own tests if you want!
+
+## Authors
+
+👤 **Isai Cespedes**
+
+- GitHub: [@Lordkaito](https://github.com/Lordkaito)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/isaicespedes)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used.
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
[OPTIONAL]
- [ ] Please make sure to personalize your MIT.md file for this project. This will make your repository more professional
![image](https://user-images.githubusercontent.com/28109626/147498740-939c2232-f97f-406f-a277-72212483e1a3.png)